packages/editor/src/store/selectors.js

```diff
@@ -1102,16 +1101,8 @@ export const isPublishSidebarEnabled = createRegistrySelector(
  * @return {Array} Block list.
  */
 export const getEditorBlocks = createSelector(
-	( state ) => {
-		return (
-			getEditedPostAttribute( state, 'blocks' ) ||
-			parse( getEditedPostContent( state ) )
-		);
-	},
-	( state ) => [
-		getEditedPostAttribute( state, 'blocks' ),
-		getEditedPostContent( state ),
-	]
+	() => getBlocks(),
+	() => [ getBlocks() ]
```

Comment:
1. Ideally we deprecate this in recommend subscribing straight to the block editor store.
2. Is `createSelector` necessary here? Previously the dependency was a call to `getEditedPostAttribute( state, 'blocks' )`, which also seems a bit weird.